feedbooks.py

```diff
@@ -29,23 +32,71 @@ def extract_feed_data(self, feed, feed_url=None):
             self.improve_description(id, m)
         return metadata, failures
 
-    def rights_for_entry(self, entry):
+    @classmethod
+    def rights_uri_from_feedparser_entry(cls, entry):
```

Comment:
This method is called by code in core OPDSImporter. As the comment says, it's not reliable, and the result needs to be fixed later, but we do the best we can.